tests/openwisp2/sample_users/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import Group as AbstractGroup
+from django.db import models
+from openwisp_users.base.models import (
+    AbstractUser,
+    BaseGroup,
+    BaseOrganization,
+    BaseOrganizationOwner,
+    BaseOrganizationUser,
+)
+from organizations.abstract import (
+    AbstractOrganization,
+    AbstractOrganizationOwner,
+    AbstractOrganizationUser,
+)
+
+
+class User(AbstractUser):
+    fiscal_number = models.CharField(max_length=16, null=True, blank=True)
+
+    class Meta(AbstractUser.Meta):
+        abstract = False
+
+
+class Organization(BaseOrganization, AbstractOrganization):
+    pass
+
+
+class OrganizationUser(BaseOrganizationUser, AbstractOrganizationUser):
+    pass
+
+
+class OrganizationOwner(BaseOrganizationOwner, AbstractOrganizationOwner):
+    pass
+
+
+# Boilerplate to add Permissions Groups
+class Group(BaseGroup, AbstractGroup):
+    pass
```

Comment:
More boilerplate for our user! :-(
The problem here has been that there is no `AbstractGroup` class, and we are proxying a concrete class and it needs to be put in order to move `GroupAdmin` under the same app label as the other models.